src/actions/validation.js

```diff
@@ -0,0 +1,30 @@
+import * as ActionTypes from '../actiontypes';
+
+import { Jurisdiction, DummyValidator } from '../contracts';
+import { JURISDICTION_ADDRESS, VALIDATOR_ADDRESS } from '../constants';
+
+const ValidationActions = {
+
+  checkValidated(address) {
+    return async function (dispatch) {
+      const jurisdiction = await Jurisdiction.at(JURISDICTION_ADDRESS);
+      const isValidated = await jurisdiction.hasAttribute(address, "VALID");
+      dispatch(ValidationActions.update(address, isValidated));
+    }
+  },
+
+  update(address, isValidated) {
+    return { type: ActionTypes.VALIDATION_UPDATED, address, isValidated };
+  },
+
+  validate(address) {
+    return async function (dispatch) {
+      const validator = await DummyValidator.at(VALIDATOR_ADDRESS);
+      await validator.validate({ from: address });
```

Comment:
Is it ok to ignore its return value?

Reply:
It doesn't have a return value. It could revert the transaction if the DummyValidator was not added as a validator.
On the next line, the checkValidated is triggered to see if the UI needs to be updated. Should we do it differently here?

Reply:
Oops I thought that this method validated an address, in the sense that it checked if it was valid, not that it added the address. My bad, sorry!